internal/service/kyma/status/modules/generator/fromerror/generator.go

```diff
@@ -46,7 +47,7 @@ func GenerateModuleStatusFromError(err error, moduleName, desiredChannel, fqdn s
 	newStatus := newDefaultErrorStatus(moduleName, desiredChannel, fqdn, err)
 
 	if errorIsTemplateNotFound(err) {
-		newStatus.State = shared.StateWarning
+		newStatus.State = shared.StateError
```

Comment:
We should not change the state of the existing one, there is no change request to ask us to change the not found error to `Error` state.